### PR TITLE
Optimize the article according to the normal expression in Chinese.

### PR DESCRIPTION
**Pull request by dufu1991**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

### CODE REVIEW

File: _articles/zh-hans/finding-users.md [hunk 1]
```diff
@@ -6,119 +6,119 @@ class: finding
 order: 3
 image: /assets/images/cards/finding.png
 related:
-  - beginners
-  - building
+    - beginners
+    - building
```
Comment (ahpook) [inline _articles/zh-hans/finding-users.md:10]: This should not be included, it is only a 2 space indentation.

**Review by ahpook — CHANGES_REQUESTED** [review]
I will trust that the translations are accurate; please fix the YAML header and I will merge. Thank you for these edits!

**Comment by dufu1991** [comment]
Resubmit the new PR.

**Review by Pilarm1234 — COMMENTED** [review]
>   #@`
Duplicate of #`

**Pull request closed without merging**